### PR TITLE
feat: message forwarder

### PR DESCRIPTION
**Pull request by noaccOS**
~Module to handle protobuf encoding/decoding and~ middleman between sockets/controllers

depends on #8 ~#13~ and includes its commits

### CODE REVIEW

**Comment by noaccOS** [comment]
Module moved to `edgehog_device_forwarder/forwarder.ex` and added some tests

File: mix.exs [hunk 1]
```diff
@@ -37,7 +37,13 @@ defmodule EdgehogDeviceForwarder.MixProject do
       {:plug_cowboy, "~> 2.5"},
       {:con_cache, "~> 1.0"},
       {:typed_struct, "~> 0.1.4"},
-      {:elixir_uuid, "~> 1.2"}
+      {:elixir_uuid, "~> 1.2"},
+      {
+        :edgehog_device_forwarder_proto,
+        git: "https://github.com/noaccOS/edgehog-device-forwarder-proto",
+        branch: "feat/elixir-gen",
+        sparse: "elixir/edgehog_device_forwarder_proto"
```
Comment (Annopaolo) [inline mix.exs]: gif

**Review by Annopaolo — CHANGES_REQUESTED** [review]

File: mix.exs (hunk 1 above)
Comment (noaccOS) [reply mix.exs]: Hasn't been merged yet ¯\\\_(ツ)_/¯

Comment (noaccOS) [reply mix.exs]: it has since been merged and the dependency has been updated to point to upstream.

**Review by rbino — APPROVED** [review]

**Pull request merged**